distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1645,7 +1641,7 @@ type Table
     make_constant_column : Any -> Column
     make_constant_column self value =
         if Table_Helpers.is_column value then Error.throw (Illegal_Argument.Error "A constant value may only be created from a scalar, not a Column") else
-            Column.from_vector_repeated value.pretty [value] self.row_count
+            Column.from_vector_repeated value.to_text value self.row_count
```

Comment:
Changed this back to `.to_text` because `pretty` was producing column names like `\'foo\'`

Reply:
I thought that was the whole point?

Isn't it just rendered as `'\'foo\''` but in fact the value being `'foo'`? i.e. you can write it using `"` as `"'foo'"`?
IMO looking from the perspective of a more 'dropdown-centric' workflow, these kinds of names for these constants seem reasonable.

But I'm not attached to this, we can keep `to_text` too if the consensus is it is better.

Reply:
I think `to_text` is fine for this - in general will be something you wouldn't be keeping as you build up stuff.